src/components/userLists/listItem.jsx

```diff
@@ -0,0 +1,22 @@
+import ProfileCircle from "../profileCircle";
+
+const ListItem = ({ user, getInitials, currentUser, renderTeacherContent, renderStudentContent, name }) => {
+  return (
+    <li key={user.id} className='students-user-card'>
+      <ProfileCircle
+        initials={getInitials(user.firstName, user.lastName)}
+        hasCascadingMenu={false}
+      />
+
+      <div className="found-user-details">
+        <span>{`${user.firstName} ${user.lastName}`}</span>
+        <p>{`Software Developer, Cohort ${user.cohortId}`}</p>
+      </div>
+      {name === "searchResults" && currentUser.role === "TEACHER"
```

Comment:
I'm not sure I like the idea that there's a conditional render based on a string value "searchResults", what are some other ways you could approach this component?